### PR TITLE
css-two-column-layout | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**
image

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.name-and-description {
+  flex-basis: 60%;
+  margin-top: 30px;
+}
+
+.icon {
+  flex-basis: 25%;
+  width: 100%;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: These will compete with one another. Choose the one that is actually the value you want, and remove the other

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  align-self: flex-start;
+}
+
+.heading {
+  text-transform: uppercase;
+  font-size: 2.8em;
+  letter-spacing: 0.05em;
+  margin: -5px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: Avoid negative margin or padding, they are a gateway to terrible things 😬

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! A couple minor things:

File: css-two-column-layout/styles.css (hunk 2 above)
Comment (angelsleung) [reply css-two-column-layout/styles.css]: Turns out that I just had to set the margin to 0 to get the effect I wanted 🤣 Cool!

**Comment by angelsleung** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice work!! 🙌

**Pull request merged**